### PR TITLE
nsc-events-next_9_507_edit-user-role-profile-page-responsive-ui

### LINKED ISSUE #504: Chore: Creator Page Responsive UI
As a developer, I will want to:

- [x]  Adjust so that page elements on the creator page are not broken or cropped or off-screen for mobile screens
- [x]  Users shouldn't have to side-scroll to view whole page

### LINKED ISSUE #507: Chore: Edit User Role and Profile Pages Responsive UI
As a developer, I will want to:

- [x]  Adjust so that page elements on the Edit User Role and Profile pages are not broken or cropped or off-screen for mobile screens
- [x]  Users shouldn't have to side-scroll to view whole page

### PR DESCRIPTION
**Pull request by intisarosman1**
Resolves #507 

This PR ensures that the Edit User Role and Profile pages have responsive UI and adjust to mobile screen sizes well, with no cutoffs or the need to scroll horizontally.

Test this PR by right clicking the page and using Inspect elements to enter the Toggle Device Toolbar and adjust the screen sizes.

Smallest mobile screen size:
Screenshot 2024-07-05 203252

Screenshot 2024-07-05 003704

### CODE REVIEW

**Review by heosman — APPROVED** [review]
Looks good!
Edit User Role page:
image

Profile page:
Could you center the "Welcome, [Name]" heading and add a exclamation point after the name?
image

**Comment by intisarosman1** [comment]
> Looks good! Edit User Role page: image
> 
> Profile page: Could you center the "Welcome, [Name]" heading and add a exclamation point after the name? image

The welcome message is centered with an exclamation point now!

**Review by IsaacJrTypes — DISMISSED** [review]
Great job on making the UI responsive! I left a few line comments that need attention. Just want to make sure we aren't merging code that could disconnect front end of the app with the backend server (NSC-events API)

File: app/auth/sign-in/page.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import blue_vertical_nsc_logo from 'public/images/blue_vertical_nsc_logo.png'
 import white_vertical_nsc_logo from 'public/images/white_vertical_nsc_logo.png'
 import { useTheme } from "@mui/material";
 
-const URL = process.env.NSC_EVENTS_PUBLIC_API_URL || "http://localhost:3000/api";
+const URL =  "http://localhost:3000/api";
```
[the remark this answers is not in the document]
Comment (intisarosman1) [reply app/auth/sign-in/page.tsx:12]: Sorry, about that! That was an accident from when I was testing the changes locally. Let me return those back to normal.

[the remark this answers is not in the document]
Comment (IsaacJrTypes) [reply app/auth/sign-in/page.tsx]: No problem! I figured the changes weren't intentional

**Review by IsaacJrTypes — APPROVED** [review]
Thank you for making the changes I requested 😃

**Pull request merged**